control-plane/PROJECT

```diff
@@ -142,4 +142,13 @@ resources:
   kind: GatewayPolicy
   path: github.com/hashicorp/consul-k8s/control-plane/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1beta1
+    namespaced: true
+  controller: true
+  domain: consul.hashicorp.com
+  group: auth
+  kind: TrafficPermissions
+  path: github.com/hashicorp/consul-k8s/control-plane/api/v2alpha1
+  version: v2alpha1
```

Comment:
nit: looks like Iryna updated versions in core to `v2beta1` rather than alpha so we should match